src/dataflow/components/nodes/controls/dropdown-list-control.tsx

```diff
@@ -15,30 +16,86 @@ export class DropdownListControl extends Rete.Control {
     };
     const handlePointerMove = (e: any) => e.stopPropagation();
 
-    this.component = (compProps: { value: any; onChange: any; optionArray: any; }) => (
-      <select
-        value={compProps.value}
-        onChange={handleChange(compProps.onChange)}
-        onPointerMove={handlePointerMove}>
-        {compProps.optionArray.map((val: any, i: any) => (
-          <option key={i} value={val}>
-            {val}
-          </option>
-        ))}
-      </select>
+    this.component = (compProps: {
+                                    value: any;
+                                    onItemClick: any;
+                                    onListClick: any;
+                                    showList: any
+                                    optionArray: any;
+                                    listClass: any;
```

Comment:
It's a preexisting condition, so arguably not part of this PR, but surely we can do better than `any` for typing the properties of our own components.

Reply:
Added some changes to fix this.